src/NTH.Windows.Forms/NTH.Windows.Forms/PlaceholderTextbox.cs

```diff
@@ -33,9 +33,14 @@ public PlaceholderTextbox()
 
         private void UpdatePlaceholder()
         {
-            var currentHandle = Handle;
-            if (currentHandle != IntPtr.Zero)
-                NativeMethods.SendMessage(currentHandle, WindowMessage.EM_SETCUEBANNER, IntPtr.Zero, _placeholder);
+            if (IsHandleCreated)
+                NativeMethods.SendMessage(Handle, WindowMessage.EM_SETCUEBANNER, new IntPtr(1), _placeholder);
```

Comment:
Yep, I also thought about something like that. I will implement that in the PR. ;)